nano/lib/blocks.cpp

```diff
@@ -991,7 +1008,8 @@ hashables (error_a, stream_a)
 }
 
 nano::state_block::state_block (bool & error_a, boost::property_tree::ptree const & tree_a) :
-hashables (error_a, tree_a)
+hashables (error_a, tree_a),
+epoch_m (nano::epoch::unspecified)
```

Comment:
Rather than repeat this default value in all constructors (and risk missing one in the future) you could add it as an in-class member initializer.